docs/en/latest/plugins/file-logger.md

```diff
@@ -128,3 +129,44 @@ curl http://127.0.0.1:9180/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
   }
 }'
 ```
+
+## Configure the matches parameter
```

Comment:
This block should be put before `Disable Plugin`

Reply:
The section should be `Filter logs`

Reply:
fix~